guides/development/hack_on_the_trento.md

```diff
@@ -1,82 +1,111 @@
-# Hack on the trento
-
-- [Requirements](#requirements)
-- [Install dependencies](#install-dependencies)
-- [Development environment](#development-environment)
-- [Setup trento](#setup-trento)
-- [Start trento in the repl](#start-trento-in-the-repl)
-- [Environment Variables](#environment-variables)
-- [Scenario loading with photofinish](#scenario-loading-with-photofinish)
+# Hack on the Trento Web
 
 ## Requirements
 
-In order to run the trento web, you need
+In order to run the Trento Web application, you need to have the following software installed on your system:
+
+1. [Elixir](https://elixir-lang.org/) - a dynamic, functional language designed for building scalable and maintainable applications.
+2. [Docker](https://docs.docker.com/get-docker/) - a platform for developing, shipping, and running applications in containers
+3. [Docker Compose](https://docs.docker.com/compose/install/) - a tool for defining and running multi-container Docker applications.
 
-1. [Elixir](https://elixir-lang.org/)
-2. [Erlang OTP](https://www.erlang.org/)
-3. [Docker](https://docs.docker.com/get-docker/)
-4. [Docker Compose](https://docs.docker.com/compose/install/)
+**Note** that you also need to have Git installed on your computer to clone the Trento repository.
```

Comment:
Not saying that it is wrong, but I find this too detailed hehe
I personally wouldn't be that informative. We expect from users to know certain things.
But personal opinion of course